centralstore/redis_store.go

```diff
@@ -1091,32 +1096,6 @@ func (t *traceStateProcessor) traceIDsByState(ctx context.Context, conn redis.Co
 	return results, nil
 }
 
-func (t *traceStateProcessor) traceIDsWithTimestamp(ctx context.Context, conn redis.Conn, state CentralTraceState, traceIDs []string) (map[string]time.Time, error) {
```

Comment:
How come we don't need this anymore, was it not being called?

Reply:
it was a left over code from a previous change that i forgot to remove